examples/create_paymail_as_admin/create_paymail_as_admin.go

```diff
@@ -0,0 +1,47 @@
+package main
+
+import (
+	"context"
+	"log"
+
+	wallet "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet-go-client/commands"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples"
+	"github.com/bitcoin-sv/spv-wallet-go-client/examples/exampleutil"
+	"github.com/bitcoin-sv/spv-wallet-go-client/internal/api/v1/querybuilders"
+	"github.com/bitcoin-sv/spv-wallet-go-client/walletkeys"
+)
+
+func main() {
+	ctx := context.Background()
+	keys, err := walletkeys.RandomKeys()
```

Comment:
The key thing I liked in the `v1` client is that all examples used the defined keys in some file (in that case it's `example_keys.go`.

Thanks to that, everyone could use this example for actual `xPub` / `paymail`.

In this new approach, all keys and paymails are randomized - and I think - crucial functionality/usability of such examples are missed.

Reply:
Yes you're right. I forgot to replace it 😞